packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -657,7 +703,10 @@ List<Variable> _createVariablesForFields(
 // TODO(jacobr): consider a new class name. This class is more just the data
 // model for a tree of Dart objects with properties rather than a "Variable".
 class Variable extends TreeNode<Variable> {
-  Variable._(this.name, ref, this.text) : _ref = ref {
+  Variable._(this.name, ref, this.text, {int offset, int childCount})
```

Comment:
make `name`, `ref`, and `text` required and named. Also, why doesn't ref have a type?

Reply:
I've added a TODO to follow this up with that change, since it looks like it would be a bit of a refactor from what we currently have and I would rather keep that refactor separate. Let me know if this is okay!

Reply:
sgtm